### ISSUE TITLE
Service Letter does not pass Accessibility test.

### ISSUE THREAD

**Issue by Igor-Ino** [LibraryOfCongress/concordia#1442]
**What behavior did you observe? Please describe the bug**
When I am running Adobe Acrobat accessibility test, some features don't pass it.

**How can we reproduce the bug?**

1. Go to 'User Profile'
2. Click on 'Download Letter'
3. Open downloaded file.
4. Run Accessibility Report

**What is the expected behavior?**
All features pass Accessibility Report

**Got screenshots? This helps us identify the issue**
Accessibility report is attached
[Letter Accessibility Report.pdf](https://github.com/LibraryOfCongress/concordia/files/6467554/Letter.Accessibility.Report.pdf)

**Desktop (please complete the following information):**

-   OS: [OS X]
-   Browser [chrome, safari]

**Additional context**

**Comment by Igor-Ino** [comment]
In addition to Accessibility Report, there is another important issue: missing metatags. To see all metadata, in Adobe Acrobat use shortcut Command-D (in Windows CTRL-D) and see all field in Description are empty (Title, Subject, Author, Keywords)

**Comment by Igor-Ino** [comment]
Created new ticket for most critical accessibility errors:

#1539 Service letter accessibility issue – primary language is not set up
#1540 Service letter accessibility issue – title is not set up 

This ticket #1442 is closed
